metadata.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Canonical
+# See LICENSE file for licensing details.
+name: hello-juju
+description: |
+  An introductory charm using the Charmed Operator Framework. Deploys a simply
```

Comment:
Typo: simply -> simple

Reply:
Nice catch :)

Reply:
Funny story: I fairly regularly (as I suppose most of us do) get recruiting emails from various companies. I often respond "Thanks for reaching out, however, I'm not *interesting* right now" because my fingers accidentally type *interesting* instead of *interested*. Though I'm probably not very interesting right now either... :-)